configure.ac

```diff
@@ -656,6 +654,7 @@ expand_path_option bindir
 expand_path_option sbindir
 expand_path_option libexecdir
 expand_path_option datadir
+expand_path_option datarootdir
```

Comment:
> directories such as datadir are defined in terms of [datarootdir]

Does that mean `datarootdir` should get expanded first? Presumably it makes no actual difference for our purposes.

(You'll notice a pattern... I'd rather ask potentially dumb questions than go learn Autoconf.)

Reply:
Ah yes, it should